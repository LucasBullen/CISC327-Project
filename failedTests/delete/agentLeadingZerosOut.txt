2c2
< Logged into agent mode.
---
> Logged into atm mode.
4c4
< Incorrect account number format (8 digits, no leading zeros). Delete cancelled.
---
> Incorrect account number format (8 digits, no leading zeros).
