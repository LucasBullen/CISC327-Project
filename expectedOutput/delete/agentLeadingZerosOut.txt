Select mode: atm or agent.
Logged into atm mode.
Enter account number to delete.
Incorrect account number format (8 digits, no leading zeros).
Logged out.
